### DISCUSSION TITLE
Save an Image present in the navigator session inside the Sandbox shared Library

### DISCUSSION THREAD

**Discussion by Gabenny** [SalesforceCommerceCloud/sfcc-ci#411] [Team Posts]
Hello,
I would like to know if it's possible to save an image present in the navigator session and store it inside  the shared library ?

Thank you in advance for your responses

**Comment by zlatinz** [comment]
Hello,
Can you elaborate a bit what you mean by "navigator session"?
Have you seen this article in the documentation https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/ImportExport/UsingaPipelinetoPOSTdata.html ?

**Comment by Gabenny** [comment]
Hello,

Thank you for your reply.

I have a single page application that can store an image inside the browser's session.
The single page application contains a button. When I click on this button I need to send the image to a SFCC controller that import this image inside my sandbox's shared library.
I hope it's clearer now.
Thank you in advance for your reply.
best regards.

**Comment by zlatinz** [comment]
Make the sending of the image as a POST request with multipart/form-data and this should make the approach from the above-mentioned documentation article applicable.

Check if this stackoverflow article helps you with the client-side portion of the solution https://stackoverflow.com/questions/46932136/uploading-files-in-single-page-applications
